### PR TITLE
feat(web-rideshare): implement UI for ride request list

### PR DESCRIPTION
**Pull request by Rediet-Ferew**
In this branch the UI for ride request list for the admin dashboard is implemented

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "prettier": "prettier --write ."
   },
   "dependencies": {
+    "@react-google-maps/api": "^2.18.1",
```
Comment (kkstarkk) [inline package.json:15]: what's this for??

File: src/components/admin/ride-request/RideRequestList.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+  const handleSelect = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    const value = event.target.value;
+    setSelectedValue(value);
+  };
+
+  return (
+    <div>
+      <div className="flex w-full justify-evenly my-10 items-center">
+        <DropDown
```
Comment (kkstarkk) [inline src/components/admin/ride-request/RideRequestList.tsx:38]: the search Bar Should come first

File: src/store/api/index.ts [hunk 3]
```diff
@@ -22,7 +23,21 @@ export const apiSlice = createApi({
...
               body: credentials,
             }),
+
+            
           }),
+          getAllRideRequests: builder.query<RideRequest[], void>({
+            query: () => '/ride-requests',
+          }), 
+      
+        searchRequestsByName: builder.query<RideRequest[], string>({
```
Comment (kkstarkk) [inline src/store/api/index.ts:33]: searchRequestsByName and filterByStatus are provided as a single endpoint..you should combine the endpoints into one

**Review by kkstarkk — CHANGES_REQUESTED** [review]

File: src/components/admin/ride-request/RideRequestList.tsx (hunk 2 above)
Comment (Rediet-Ferew) [reply src/components/admin/ride-request/RideRequestList.tsx:38]: the ui is the drop down first should I change it?

**Review by kkstarkk — APPROVED** [review]

**Pull request merged**